File: gluecodium/src/test/resources/smoke/errors/output/android/com/example/smoke/SomeTypeCollection.java
/*

 *
 */

package com.example.smoke;


public final class SomeTypeCollection {
    public enum SomeTypeCollectionError {
        ERROR_A(0),
        ERROR_B(1);
        /**
         * @hidden
         */
        public final int value;

        SomeTypeCollectionError(final int value) {
            this.value = value;
        }
    }
    public static final class SomeException extends Exception {
        /**
         * @hidden
         * @param error The error.
         */
        public SomeException(final SomeTypeCollection.SomeTypeCollectionError error) {
            super(error.toString());
            this.error = error;
        }

        /**
         * @hidden
         */
        public final SomeTypeCollection.SomeTypeCollectionError error;
    }


}

File: gluecodium/src/test/resources/smoke/errors/output/android-kotlin/com/example/smoke/SomeTypeCollection.kt
/*

 *
 */

@file:JvmName("SomeTypeCollection")

package com.example.smoke


class SomeTypeCollection {

    enum class SomeTypeCollectionError(@JvmField val value: Int) {
        ERROR_A(0),
        ERROR_B(1);
    }
    class SomeException(@JvmField val error: SomeTypeCollection.SomeTypeCollectionError) : Exception(error.toString())









}

